x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts.ts

```diff
@@ -361,11 +362,22 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     });
 
     it('should delete all selection', async () => {
```

Comment:
nit: Seems like testing for this could be covered in a unit test instead of end to end.

Reply:
You can test the implementation of the fix in a unit test, but you can't test the actual behaviour as it relies on hooks and calls being made inside of other components (including EUI).
You could never be sure it's working correctly with just a unit test...

I have a real fear that we're compromising on test quality instead of addressing the scaling issues in our infrastructure. :/